.github/workflows/test.yml

```
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, ubuntu-20.04, macos-12, macos-11, windows-2022, windows-2019]
+        os: [ubuntu-22.04, ubuntu-20.04, macos-13, macos-12, macos-11, windows-2022, windows-2019]
```

Don't we miss the arm64 runner here?
```suggestion
        os:
          - ubuntu-22.04
          - ubuntu-20.04
          - macos-13-1x
          - macos-13
          - macos-12
          - macos-11
          - windows-2022
          - windows-2019
```

---

do you mean the `xl` version?

my understanding was mac arm64 was only supported for self-hosted runners

---

I see, misread the label.